docs/notebooks/proton_radiography_particle_tracing_custom_source.ipynb

```diff
@@ -0,0 +1,598 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Synthetic Radiographs with Custom Source Profiles"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[SyntheticProtronRadiography]: ../api/plasmapy.diagnostics.proton_radiography.SyntheticProtonRadiograph.rst#plasmapy.diagnostics.proton_radiography.SyntheticProtonRadiograph\n",
+    "\n",
+    "An experimental reality of charged particle radiography experiments is that the source of the particles has finite size and often generates a non-uniform angular velocity distribution. For this reason, the \n",
+    "[SyntheticProtronRadiography] particle tracing algorithm allows users to specify their own initial particle positions and velocities. This example will demonstrate how to use this functionality to create a more realistic synthetic radiograph that includes these source profile effects."
```

Comment:
```suggestion
    "In the real world the particle source for charged particle radiography experiments is finite in size and often has a non-uniform angular velocity distribution. For this reason, the [SyntheticProtronRadiography] particle tracing algorithm allows users to specify their own initial particle positions and velocities. This example will demonstrate how to use this functionality."
```

I still find this lead in a bit clunky.  Every time I get to the middle of the first sentence I have to re-scan the sentence to determine what is trying to be said.

Reply:
I agree, how about this 

In real charged particle radiography experiments, the finite size of the particle source limits the resolution of the radiograph. Realistic sources may also generate a non-uniform angular distribution of particles, superimposing a large scale non-uniform "source profile" on the radiograph. For these reasons, the 
[SyntheticProtronRadiography] particle tracing algorithm allows users to specify their own initial particle positions and velocities. This example will demonstrate how to use this functionality to create a more realistic synthetic radiograph that includes these finite source and source profile effects.